xbmc/cores/VideoPlayer/DVDDemuxers/DVDFactoryDemuxer.cpp

```diff
@@ -60,20 +60,6 @@ CDVDDemux* CDVDFactoryDemuxer::CreateDemuxer(std::shared_ptr<CDVDInputStream> pI
       return nullptr;
   }
 
-  bool streaminfo = true; /* Look for streams before playback */
-  if (pInputStream->IsStreamType(DVDSTREAM_TYPE_PVRMANAGER))
-  {
-    /* Don't parse the streaminfo for some cases of streams to reduce the channel switch time */
-    bool useFastswitch = URIUtils::IsUsingFastSwitch(pInputStream->GetFileName());
```

Comment:
I think URIUtils::IsUsingFastSwitch can be deleted too

Reply:
Done